docker-compose.yaml

```diff
@@ -0,0 +1,125 @@
+version: '3'
+
+services:
+  db:
+    container_name: wandb_db
+    image: mysql:8.0
```

Comment:
Are we fully on 8.0? I know EOL is coming up, but can't remember where we are on this